src/main/java/com/powsybl/caseserver/CaseController.java

```diff
@@ -224,4 +225,13 @@ public ResponseEntity<List<CaseInfos>> getMetadata(@RequestParam("ids") List<UUI
         LOGGER.debug("get Cases metadata");
         return ResponseEntity.ok().body(caseService.getMetadata(ids));
     }
+
+    @GetMapping(value = "/cases/caseBaseName")
```

Comment:
the value attribute in @GetMapping is redundant when there’s only one parameter
remove it
```suggestion
    @GetMapping("/cases/caseBaseName")
```